janusgraph-hbase/src/test/java/org/janusgraph/graphdb/hbase/HBaseOperationCountingTest.java

```diff
@@ -42,4 +43,10 @@ public void testCacheConcurrency() {
         //Don't run this test;
     }
 
+    @Override
+    @RepeatedIfExceptionsTest(repeats = 3)
+    public void testKCVSAccess1(){
```

Comment:
@farodin91 Do you remember why you marked this particular test as flaky? I can't find any issue which mentions this test as being flaky.
But I'm mainly asking because this test still failed in two different runs for this PR, for Java 8 and 11 and it looks like just retrying the test can't work here:

```
expected: <6> but was: <24>
```

The was executed 4 times and the check uses a metric that has the same name and is registered with the same graph so it looks to me like repeating the test actually leads to the failures here.

Reply:
@farodin91 and I talked about this directly and he wasn't sure any more why he had added this in the first place as that was over half a year ago so we agreed to remove this again from this PR.